tasks/Functions-client/buildOffchainSecrets.js

```diff
@@ -8,40 +9,35 @@ task(
   "functions-build-offchain-secrets",
   "Builds an off-chain secrets object that can be uploaded and referenced via URL"
 )
-  .addOptionalParam("output", "Output file name (defaults to offchain-secrets.json)")
```

Comment:
You can add a default output file name here instead of using a ternary operator below.

Reply:
changed to ...(defaults to offchain-encrypted-secrets.json)")